backends/ebpf/targets/xdp_target.py

```diff
@@ -0,0 +1 @@
+/home/mbudiu/git/p4c/extensions/p4c-xdp/xdp_target.py
```

Comment:
This seems unrelated to the PR and also references a file in @mbudiu-vmw's machine...